activemodel/test/cases/type/immutable_string_test.rb

```diff
@@ -17,6 +17,26 @@ class ImmutableStringTest < ActiveModel::TestCase
         assert_same s, type.cast(s)
         assert_same s, type.deserialize(s)
       end
+
+      test "leaves validly encoded strings untouched" do
+        s = "string with àccénts".encode("iso-8859-1")
```

Comment:
```suggestion
        s = "string with àccénts".encode(Encoding::ISO_8859_1)
```

Personal pet peeve of mine, I absolutely hate that encodings can be passed as strings.

Reply:
I fully respect that pet peeve! PR updated accordingly.